modules/internal/ChapelSyncvar.chpl

```diff
@@ -316,12 +316,11 @@ module ChapelSyncvar {
 
   inline operator :(from, type t:_syncvar)
```

Comment:
Looks like you removed the compilerWarning but accidentally didn't add a `deprecated` keyword to this operator?

Reply:
This was intentional. With the deprecated keyword I was seeing warnings for patterns like `var x: sync int = 1` and we do intend to support that. Because of the "4 levels of conversions", since that pattern is allowed, we also need to allow `1: sync int` (although I haven't been able to get the compiler to call this code in that case).